src/components/header/Header.css

```diff
@@ -1,3 +1,84 @@
 * {
   box-sizing: border-box;
 }
+
+.Bookstore-CMS {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+a {
+  color: #121212;
+}
+
+a:active {
+  color: #121212;
+}
+
```

Comment:
- [ ] Hi @getssh, Great effort working on the styling of the bookstore project, But currently i would suggest you work on some alignment in your project. The `Bookstore CMS` logo and the `icon` align with the books and form vertically from top to bottom as seen in the Zeplin Design. 

Below is the difference spotted 👇 

| Your Design 👇 | Zeplin Design 👇 |
| ------------- | ------------- |
| ![tam](https://user-images.githubusercontent.com/78634841/230743488-3c74294a-cc3c-444d-836f-91f7ef03291c.png) | ![org](https://user-images.githubusercontent.com/78634841/230743482-b9a43234-1d10-4f52-91b1-8175fc307bdd.png) |

Reply:
Hello @brainconnect93, Thank you for the review. I have implemented the requested change